resources/rules/post_identifier_merge/temp/irefweb/irefweb_binary_interactions.clj

```diff
@@ -47,15 +47,18 @@
         
         ;; select the other interactor
         (?/super_record obo/BFO_0000051 ?/interactor_record1) ; BFO:has_part
+        (?/super_record obo/BFO_0000051 ?/interactor_record2) ; BFO:has_part
 
         ;; get the bioentity (protein) corresponding to interactor 1
+        (?/interaction_record obo/BFO_0000051 ?/interaction_record1) ; BFO:has_part
```

Comment:
I'm not quite sure about an ?/interaction_record having-part another ?/interaction_record1. Perhaps the second variable should be named differently?

If I remember correctly, each IRefWeb 'super' record has part 2 interactor records and 1 interaction record. This rule might be better debugged against a live triple store given its complexity.